cassandra.go

```diff
@@ -45,10 +45,18 @@ type CassandraConfig struct {
 	DisableInitialHostLookup bool     `json:"disableinitialhostlookup"`     // Don't preform ip address discovery on the cluster, just use the Nodes provided
 	PreferRPCAddress         bool     `json:"prefer_rpc_address"`           // Prefer to connect to rpc_addresses during cluster discovery
 
+	// SSL Options
+	Ssl CassandraSslConfig `json:"ssl"` // ssl options cert/key/ca ...
+
 	// TestMode affects whether a keyspace creation will be attempted on Cassandra initialization.
 	TestMode bool `config:"optional"`
 }
 
+type CassandraSslConfig struct {
+	Enabled bool
+	Options *gocql.SslOptions
+}
+
 func (c CassandraConfig) String() string {
 	return fmt.Sprintf("CassandraConfig(DataCenter=%v, Nodes=%v, Keyspace=%v, ReadConsistency=%v,"+
```

Comment:
would it add value to add `SSL.Enabled=%v` to the `String()` print ?

Reply:
yes